SE-class-2021

```diff
@@ -118,3 +118,4 @@ xxxxxxx, 1f0c7a8033bd2f83b3da4b38a2b575ba35e2451033c39faf85eeb8b25d85a331(sha256
 2211313, Junya Yamamoto
 2211258, Yue Fujiki
 2211313,Junya Yamamoto
+2211218
```

Comment:
The document instructed us to add it to the `SE-class-2021`, is this a mistake for `SE-class-2022`?

If so, I will correct it.

Reply:
you can submit to either. The key point is just to make a pull request

Reply:
> you can submit to either. The key point is just to make a pull request

I understood, Thank you for your reply :)